scripts/publish_cratesio_new_versions.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+set -eu
+
+cargo install cargo-get
+workspace_members=$(cargo get --delimiter ";" workspace.members)
+current_dir="$(pwd)"
+
+IFS=';' read -a array <<< "$workspace_members"
+
+for workspace_member in "${array[@]}" 
+do
+    crate_dir="$current_dir/$workspace_member";
+
+    package_name=$(cargo get --entry $crate_dir package.name)
+    package_version=$(cargo get --entry $crate_dir package.version)
+    cratesio_version=$(cargo search $package_name --limit 1 | head -n 1 | sed -n 's/'$package_name' = "\([^"]*\)".*/\1/p');
+
+    # if local version is different from crates.io version, then publish
+    if [ $package_version != $cratesio_version ]; then
+        echo "Local version of $package_name is $package_version, while the one on crates.io is $cratesio_version";
+        # Publish to crates.io
+        cargo publish -p $package_name
```

Comment:
Do we need some credentials?

Reply:
Oh yes, forgot to attach an API_KEY

Reply:
[aa9c843](https://github.com/0xSpaceShard/starknet-devnet-rs/pull/332/commits/aa9c84300870915e7fe4e7b11fc8014e31747062)

Reply:
Did you add a CircleCI variable?